celer/cython_utils.pyx

```diff
@@ -399,6 +402,9 @@ cpdef floating dnorm_l1(
         else:
             Xj_theta = fdot(&n_samples, &theta[0], &inc, &X[0, j], &inc)
 
+        if l1_ratio != 1:
```

Comment:
add a comment here: "minus sign to account for the choice theta = y - Xw and not theta = Xw -y"